internal/cloudapi/v2/v2.go

```diff
@@ -205,6 +205,22 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 		}
 	}
 
+	if request.Customizations != nil && request.Customizations.Filesystem != nil {
+		var fsCustomizations []blueprint.FilesystemCustomization
+		for _, f := range *request.Customizations.Filesystem {
+
+			fsCustomizations = append(fsCustomizations,
+				blueprint.FilesystemCustomization{
+					Mountpoint: f.Mountpoint,
```

Comment:
The reason this does not compile is that `f.Mountpoint` is `*string`, but `Mountpoint` expects `string`.

Either we need to change `(v2.Filesystem).Mountpoint` to `string` too, or we must dereference it like `Mountpoint: *f.Mountpoint`. However, if we do that, we better check that it is actually set first. As mentioned in the earlier comments, I think the correct solution is to change `(v2.Filesystem).Mountpoint` to `string` (by making it `required` in the open api spec).